tests/integ/test_huggingface.py

```diff
@@ -111,17 +101,12 @@ def test_huggingface_training(
 
 
 @pytest.mark.release
-@pytest.mark.skipif(
-    integ.test_region() in integ.TRAINING_NO_P2_REGIONS
-    and integ.test_region() in integ.TRAINING_NO_P3_REGIONS,
-    reason="no ml.p2 or ml.p3 instances in this region",
-)
 @pytest.mark.skip(
     reason="need to re enable it later t.corp:V609860141",
 )
 def test_huggingface_training_tf(
     sagemaker_session,
-    gpu_instance_type,
+    gpu_pytorch_instance_type,
```

Comment:
This will fail since `gpu_pytorch_instance_type` fixture is doing validation for PyTorch `fw_version` check which can't be used for Tensorflow test. Can you write another fixture to get right GPU instance_type ?

Reply:
Keeping to `gpu_instance_type` for now, we will revisit the defination once we are ready to support newer TF huggingface images.